src/Gui/View3DInventorExamples.cpp

```diff
@@ -184,7 +184,7 @@ void doClipping(SbVec3f trans, SbRotation rot)
 
 void draggerCB(void * /*data*/, SoDragger * dragger)
 {
-  SoTransformerDragger * drag = (SoTransformerDragger *)dragger;
+    auto drag = (SoTransformerDragger *)dragger;

```

Comment:
Spacing.

Reply:
Surrounding code has 2chr indent.

Reply:
Yes, sorry - my point is that the surrounding code has 2 character indent, and your new code does not.

Reply:
done